src/components/TipCard/TipCard.js

```diff
@@ -1,23 +1,35 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './TipCard.css';
 import PropTypes from 'prop-types';
 import Rating from 'react-rating';
 
 const Tip = ({ rating, id, title, description, mod, date, handleDelete, handleRating}) => {
   
+  const [message, setMessage] = useState('');
+  let timer;
+
+  const onRating = (value, id) => {
+    handleRating(value, id);
+    setMessage('You have successfully rated this tip')
+    timer = setTimeout(() => setMessage(''), 3000)
+  }
+
   return (
     <article className='tip-card'>
-      <div className={`styling-box mod-${mod}`}></div>
+      <div className={`styling-box mod-${mod}`}>
+      <p className="mod-text">Mod {mod}</p>
+      </div>
       <h2>{title}</h2>
       <hr />
       <p className="description">{description}</p>
-      <div className='details'>
-      <p>Mod: {mod}</p>
-      <p>Your Rating: <Rating onClick={(value) => handleRating(value, id)}
+      <div className='rating-details'>
+      <p>Your Rating:</p>
+      <p><Rating onClick={(value) => onRating(value, id)}
```

Comment:
Nice addition of making it a helper function to call back the handlerating and add a message.